config/locales/en.yml

```diff
@@ -944,6 +944,11 @@ en:
     no_edits: There are no recent edits.
     plagiarism_report: Plagiarism Report
     possible_plagiarism: Possible Plagiarism
+    plagiarism_explanation: >
+      These reports are not 100% accurate. For example, a direct quote with proper attribution
```

Comment:
I suggest adding this at the beginning: "Wikipedia's plagiarism detection system uses Turnitin/iThenticate to compare new edits with published text."